Automattic/woocommerce-payments
# Add Disputes filters

Fixes #3591

#### Changes proposed in this Pull Request

Add Date and Status filters to the Disputes list table.
<img width="850" alt="Screenshot 2022-01-13 at 12 09 58" src="https://user-images.githubusercontent.com/8667118/149310202-7477c014-b11f-4045-97d3-194b8f2e1fee.png">

#### Testing instructions

1. Add a number of orders with disputes having different statuses using [Stripe's test cards to pay for the orders](https://stripe.com/docs/testing#disputes). Using one of these cards will generate a dispute after the order is paid.
2. Note that this PR relies on the [pagination and filters functionality](https://github.com/Automattic/woocommerce-payments-server/pull/1464).
3. Follow instructions from 1408-gh-Automattic/woocommerce-payments-server to have disputes saved in the cache.
4. Observe that the filters controls appear at the top of the table.
5. Check that the filter controls work as expected.
6. Observe that the correct total number of Disputes is displayed at the bottom of the table when filters are applied.

-------------------
